alpine/Dockerfile

```diff
@@ -16,7 +16,10 @@ LABEL vendor="Intelsdi-X" \
 
 EXPOSE 8181
 
-RUN apk add --update curl && \
+RUN apk update && \
+    apk add curl && \
+    apk add bash && \
+    apk add jq && \
```

Comment:
You're correct in the influxdb large test I choose to use a combination of snapctl with awk and grep to test the state of the task.   I feel in other large tests we may want to use curl in which case verifying the result will be made easier if `jq` was available.